python/ql/src/Security/CVE-2018-1281/BindToAllInterfaces.qhelp

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>Sockets can be used to communicate
+with other machines on a network.
+You can use the (IP address, port) pair
+to define the access restrictions for the socket you create.
+When using the built-in Python <code>socket</code> module
+(for instance, when building a message sender service
+or an FTP server data transmitter),
+one has to bind the port to some interface.
+When you bind the port to all interfaces
+using <code>0.0.0.0</code> as the IP address,
+you essentially allow it to accept connections from any IPv4 address
+provided that it can get to the socket via routing.
+Binding to all interfaces is therefore associated with security risks.</p>
+</overview>
+
+<recommendation>
+<p>Bind your service incoming traffic only to a dedicated interface.
+If you think you need to bind more than one interface
```

Comment:
Suggest removing "think" it makes it sound as if we don't think there's ever a good reason for doing this.
```suggestion
If you need to bind more than one interface
```